helixbass/tree-sitter-lint
# Hard-coded "project-local binary"

In this PR:
- as part of trying to play with the idea of (instead of dynamically-linking or using WASM to load plugins at runtime) generating a project-local "custom binary", manually create a hard-coded project-local Rust project and invoke its release build

To test:
The path to the project-local binary is currently hard-coded (in `src/cli.rs`) so you would want to change it to wherever your corresponding local release build would be
And you would need to manually trigger creating that release build (eg by running `cargo build --release` from `.tree-sitter-grep/tree-sitter-grep-local/`)
At that point if you run this project eg via `cargo run` it should work (the same as before, using the hard-coded rules)

Based on `fixing-tests`
